src/main/java/org/fcrepo/camel/processor/ProcessorUtils.java

```diff
@@ -115,5 +126,24 @@ public static String insertData(final String serializedGraph, final String named
         query.append("}");
         return query.toString();
     }
+
+    /**
+     * Tokenize a property placeholder value
+     *
+     * @param context the camel context
+     * @param property the property name
```

Comment:
@birkland, I updated the Javadoc to be more explicit. Thanks for the suggestion!